integration_test/ops_agent_test.go

```diff
@@ -3898,12 +3907,11 @@ func installGolang(ctx context.Context, logger *logging.DirectoryLogger, vm *gce
 			Start-Process msiexec.exe -ArgumentList "/i","golang.msi","/quiet" -Wait `, goVersion, goArch)
 	} else {
 		installCmd = fmt.Sprintf(`
-			set -e
 			gsutil cp \
 				"gs://stackdriver-test-143416-go-install/go%s.linux-%s.tar.gz" - | \
-				tar --directory /usr/local -xzf /dev/stdin`, goVersion, goArch)
+				sudo tar --directory /usr/local -xzf /dev/stdin`, goVersion, goArch)
```

Comment:
RunScriptRemotely runs with "sudo" while RunRemotely does not, so I had to stick a sudo on here.